pages/api/benefits.js

```diff
@@ -0,0 +1,72 @@
+import { getBenefits } from "../../lib/benefits";
+
+export default async (req, res) => {
+  if (req.method === "GET") {
+    //Get situation cookie
+    let situation = {};
+    if (req.cookies.situation) {
+      situation = JSON.parse(req.cookies.situation);
+    }
+
+    //Create list of benefits and set eligibility
+    let benefits = await getBenefits("en");
```

Comment:
You should be able to get the language from the request so this doesn't need to be hardcoded to "en".

Reply:
Language is now from the accept-language header